tidy3d/web/api/container.py

```diff
@@ -354,15 +366,17 @@ def load_sim_data(self, task_name: str) -> SimulationDataType:
         task_data_path = self.task_paths[task_name]
         task_id = self.task_ids[task_name]
         web.get_info(task_id)
+
         return web.load(
             task_id=task_id,
             path=task_data_path,
             replace_existing=False,
-            verbose=self.verbose,
+            verbose=False,
```

Comment:
I don't understand why this must be false now… does it interact with the multi-threaded processes somehow?

Reply:
maybe I should add a log here, though?
```py
if self.verbose:
    console.log("Loading {N} tasks from files.")
```

Reply:
I tried this out and it was a bit clunky maybe. Do we want to log something every time a user iterates through batch data? Not sure what the user gains from this. what do you think?

```py
for task_name, sim_data in batch_data.items():  # <- would log
    ...
```

Reply:
When I did the similar multi threading script, I think that the original ogging caused some bugs on either downloading all tasks or sometimes just a section of them. Removing the original logging verbosity made it work as expected. However, maybe this implementation sorts that issue under the hood properly.